test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13575,6 +13577,22 @@ def create_script(self, body=None, uid=None, gid=None, hostname=None):
         if isinstance(body, list):
             body = '\n'.join(body)
 
+        if self.platform == 'cray' or self.platform == 'craysim':
+            body = body.split("\n")
+            for line in body:
+                if line.startswith("#PBS") and "select=" in line:
+                    line_index = body.index(line)
+                    line_arr = line.split(" ")
+                    for j in line_arr:
+                        if j.startswith("select="):
```

Comment:
This assumes someone did #PBS -l select.  What if they did #PBS -lselect? (without the space)